### ISSUE TITLE
Failed to generate GodotSharp solution. Error: 47

### ISSUE THREAD

**Issue by ghost** [godotengine/godot#12466]
Using current master, all godot projects using mono fail to run.

The CLI errors are:

```ERROR: _load: Invalid tag in doc file: description                                                                                                                                                                                                                             
   At: editor/doc/doc_data.cpp:830.                                                                                                                                                                                                                                            
ERROR: _generate_cs_type: Condition ' !setter && !getter ' is true. returned: ERR_BUG                                                                                                                                                                                          
   At: modules/mono/editor/bindings_generator.cpp:665. 
```

This started happening after I re-generated the mono glue with the current master & recompiled.

**Comment by neikeq** [comment]
This was caused by a doc regression, but tagging as a bug in the mono module since we should get those from ClassDB instead of DocData.

**Comment by neikeq** [comment]
@akien-mga I won't re-write this in time for tomorrow, so we must make sure [this regression](https://github.com/godotengine/godot/pull/12396/files#r147568324) is fixed before alpha2.

**Comment by akien-mga** [comment]
Doctool will fix it, that PR was bogus.

**Comment by neikeq** [comment]
I guess this can be closed, right?

**Comment by tagcup** [comment]
> we should get those from ClassDB instead of DocData.

If this is done or not needed anymore, sure

**Comment by neikeq** [comment]
Fixed by #15136
